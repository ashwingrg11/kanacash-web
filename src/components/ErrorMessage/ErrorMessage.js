import React, { Component } from 'react';
import COLORS from '../../assets/Color/Color'
import PropTypes from "prop-types";
class ErrorMessage extends Component {
    render() {
        return (
            <div style={{color:`${COLORS.RED_COLOR}`}}>
                {this.props.errorMessage}
            </div>
        );
    }
}
ErrorMessage.propTypes = {
    errorMessage:PropTypes.string
}

export default ErrorMessage;