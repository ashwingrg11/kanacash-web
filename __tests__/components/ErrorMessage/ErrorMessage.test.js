import React from "react";
import ErrorMessage from "../../../src/components/ErrorMessage/ErrorMessage";
// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

describe("<ErrorMessage />", () => {
  it("ErrorMessage renders correctly", () => {
    const tree = renderer.create(<ErrorMessage />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
