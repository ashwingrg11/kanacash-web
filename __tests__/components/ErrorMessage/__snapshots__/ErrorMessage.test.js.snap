// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessage /> ErrorMessage renders correctly 1`] = `
<div
  style={
    Object {
      "color": "#dc3545",
    }
  }
/>
`;
